start.sh

```
@@ -165,10 +186,13 @@ load_env_file() {
     done < "$ENV_FILE"
 }
 
-# Check if ports are available
-check_port $BACKEND_PORT "${PRODUCT_NAME} server"
-if [ "$DEBUG_MODE" = "true" ]; then
-    check_port $DEBUG_PORT "Debug server"
+# Check if ports are available. The bootstrap one-shot does not bind the server
+# port, so the check is skipped in that mode.
+if [ "$BOOTSTRAP_MODE" != "true" ]; then
+    check_port $BACKEND_PORT "${PRODUCT_NAME} server"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)